src/App.js

```diff
@@ -1,64 +1,62 @@
-import React from 'react';
+import React, { useState } from "react";
+import { useDispatch } from "react-redux";
 import './App.css';
-import {useState} from 'react';
-import FilterList from './components/FilterList';
+import FilterList from "./components/FilterList";
 
 function App() {
+    const dispatch = useDispatch();
+    const [newTask,setNewTask] = useState(null);
+    const [status,setStatus] = useState('Pending');
```

Comment:
load redux `taskList` on App.js. Then we need to filter out the tasks on basis of status, create a useEffect that will take these 3 statuses as dependency array. 
Create another state that will hold the filter out `taskslist` and pass these through to `<taskList/>` 

Primarly flow would be 
- Take tasks list from redux
- Use hook to filter out tasks on basis of status
- Passed filter tasks to `TaskList` component

Then we don't need any conditions to filter out on child component